### ISSUE TITLE
Feature Request: Easy way to copy the obsws: URI to clipboard

### ISSUE THREAD

**Issue by Rosuav** [obsproject/obs-websocket#1025]
### Feature Request Type

Settings Dialog

### Requested Feature

The QR code contains a simple encoding of all three crucial pieces of information (IP, port, password). It would be convenient to use this without an actual QR code reader, by simply allowing it to be copied to the clipboard just like the three individual components.

### Requested Feature Usage Scenario

Any WS client that isn't connected to a phone will want to get this information. By making this easy to copy/paste, you can also de facto standardize on it being the single easiest way to configure a WS client, which will be more convenient for everyone.

**Comment by sugoidogo** [comment]
Note that qr code scanning isn't exclusive to phones, but I agree that there should be an easier way for any given program to be able to connect to obs-websocket. That's why I put together [obsconnect](https://github.com/sugoidogo/obsconnect), but I think an authorization prompt like that one should be added to obs-websocket as an alternative or replacement to password authorization, since a system like that could use unique tokens per authorized app that can then be revoked on a per-app basis.

**Comment by sugoidogo** [comment]
Can I ask why asking for that permission is a big deal? I only tested OBS connect in Chrome but the access control was very fine-grained, to the point that the only thing that actually had to be shared was the window that shows the QR code, so it's not as if OBS connect could get any information beyond that.

**Comment by Rosuav** [comment]
Last I checked, getting permission for screensharing required an extension. Has that changed?

**Comment by Rosuav** [comment]
Also - it's not just about permissions. The URI can easily be shared in any way (with or without the associated password), as it's simple text. The QR code is less easy to share.

**Comment by sugoidogo** [comment]
> Last I checked, getting permission for screensharing required an extension. Has that changed?

Yes, media APIs were added along with webrtc to facilitate standardized peer to peer video calling/screen sharing

**Comment by Rosuav** [comment]
> > Last I checked, getting permission for screensharing required an extension. Has that changed?
> 
> Yes, media APIs were added along with webrtc to facilitate standardized peer to peer video calling/screen sharing

Ah okay. Still, the ability to share on any textual communication channel is a very large piece of convenience, and doing that with a single URI rather than three separate pieces (host, port, and password) will definitely make things easier.

**Comment by sugoidogo** [comment]
> Also - it's not just about permissions. The URI can easily be shared in any way (with or without the associated password), as it's simple text. The QR code is less easy to share.

I think that could be debated but I will agree that there are scenarios where copying the uri string to clipboard is easier than scanning it from the qr code
